ietf/static_src/css/iab.scss

```diff
@@ -171,3 +181,34 @@
 .iab_body {
     background-color: rgba(62, 139, 193, 0.08);
 }
+
+.dropdown-toggle__iab {
+    &:after {
```

Comment:
Minor point – it works either way, but _technically_ pseudo elements (before & after) should have double colon, i.e. `::after`

Reply:
Yep, I've changed this